libseccomp/src/notify.rs

```diff
@@ -159,6 +170,36 @@ impl ScmpNotifReq {
     ///
     /// If an issue is encountered getting a notification request,
     /// an error will be returned.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use libseccomp::*;
+    /// # use libc::{fork, getpid, syscall, SYS_getpid};
+    /// # use std::process::exit;
+    /// // Get the current process ID
+    /// let parent_pid = unsafe { getpid() };
+    /// let mut ctx = ScmpFilterContext::new(ScmpAction::Allow)?;
+    /// ctx.add_rule(ScmpAction::Notify, ScmpSyscall::from_name("getpid")?)?;
+    /// ctx.load()?;
+    /// let fd = ctx.get_notify_fd()?;
+    /// let pid = unsafe { fork() };
+    /// if pid == 0 {
+    ///     // Child process
+    ///
+    ///     // the return value of getpid() is the child process ID
+    ///     let child_pid = unsafe{ getpid() };
+    ///     // Check if the child process ID matches the the parent process ID
```

Comment:
This comment can be changed.

```
// The child PID should be same as the parent PID by the notification.
```

Reply:
@mayank-mrinal-sony 
Sorry, I fixed the comment. Could you check it again?

Reply:
No problem I'll make the changes